swarm/api/routes/wisdom.py

```
@@ -421,7 +423,13 @@ async def get_wisdom_artifacts(run_id: str):
 
     Raises:
         404: Run not found or no wisdom artifacts.
+        400: Invalid run_id.
     """
+    try:
+        validate_path_component(run_id, "run_id")
+    except ValueError as e:
+        raise HTTPException(status_code=400, detail=str(e))
```

Ruff (0.15.4)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)